packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/example/WorklistDataGridWithSelections.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import WorklistDataGrid from 'terra-worklist-data-grid';
+
+const gridDataJSON = {
+  cols: [
+    { id: 'Column-0', displayName: 'Patient', isSelectable: true },
```

Comment:
The most common usage of this component is that most of the cells will be selectable. So this probably doesn't need a new example and could instead just update the first and second ones.

Reply:
@eawww Does it make sense to just have one example that exhibits all the functionality then?

Reply:
I think as we add more complex features we can have separate examples but consolidating simple things like this seems reasonable.